src/views/v2/events-bar/filters.php

```diff
@@ -14,5 +14,10 @@
  */
 ?>
 <div class="tribe-events__events-bar-filters">
-	<button class="tribe-common-c-btn-icon tribe-common-c-btn-icon--border tribe-common-c-btn-icon--filters"></button>
+	<div class="tribe-events__events-bar-filters--search">
```

Comment:
I could be wrong here, but I don't know if this is a filter button. I think it's a button that displays/hides the search form.

Reply:
You want to rename the section?

Reply:
No, I mean the search button that displays on mobile. I think that the search button is more part of the search block rather than the filter buttons.